server/meshmodel/argo-cd/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,45 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha2",
```

Comment:
Still incorrect, please look at existing relationship def, eg: https://github.com/meshery/meshery/blob/0d2c843bb5c1093b3b1d281130689298859a495a/server/meshmodel/kubearmor/v1.2.1/v1.0.0/relationships/hierarchical_inventory.json

Reply:
Thanks for guidance. It is updated.